src/content/ui/adaptive-responsive/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Overview
```

Comment:
Readers will see this as the title of the page, so it won't be clear what it's the overview for.

```suggestion
title: Adaptive and responsive design in Flutter
```